adaptivefiltering/dataset.py

```diff
@@ -14,14 +14,30 @@ def __init__(self, filename):
             are interpreted (in this order) with respect to the directory set with :any:`set_data_directory`,
             the current working directory, XDG data directories (Unix only) and the Python package
             installation directory.
+
         :type filename: str
         """
+        # initilize warning threshold to warn the user that show() is not available
+
         filename = locate_file(filename)
-        self.data = laspy.file.File(filename, mode="r")
+        # old laspy style
```

Comment:
I would prefer outdated code to be removed from the repository. We will not move back to `laspy<2`.